### ISSUE TITLE
Refresh button on all pages

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#64]
## User story

1. As a facility manager
2. I want to have a refresh button
3. So that I can refresh all pages

## Acceptance criteria

- [x] Refresh button is inserted on the right side of the pages
- [x] When pressed the button reloads all views
- [x] After refresh all data is up to date

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release
